neutron/test/convert_test.cpp

```diff
@@ -280,8 +280,9 @@ TEST_P(ConvertTest, Wavelength_to_Tof) {
   ASSERT_TRUE(tof.contains("counts"));
   // Broadcasting is needed as conversion introduces the dependance on
   // Dim::Spectrum
-  EXPECT_EQ(tof.coords()[Dim::Tof], broadcast(tof_original.coords()[Dim::Tof],
-                                              tof.coords()[Dim::Tof].dims()));
+  EXPECT_TRUE(all(is_approx(tof.coords()[Dim::Tof],
```

Comment:
This is the precision loss problem you mentioned?

Reply:
Well, it was what I mentioned, but it turned out that it is not due to precision loss, but different (old) CODATA used in boost. By chance there seemed to be an exact double match before, but not now.

Reply:
I see you have actually been able to tighten tolerances elsewhere.